actors/puppet/puppet.go

```diff
@@ -91,6 +92,19 @@ func (a Actor) ReturnMarshalCBORFailure(rt runtime.Runtime, _ *adt.EmptyValue) *
 	return &FailToMarshalCBOR{}
 }
 
+func (a Actor) PersistStateMarshalCBORFailure(rt runtime.Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
```

Comment:
I think `RuntimeTransactionMarshalCBORFailure` would align better with the other method names in this file.